wifimgr.py

```diff
@@ -236,7 +239,11 @@ def handle_configure(client, request):
 
 
 def handle_not_found(client, url):
-    send_response(client, "Path not found: {}".format(url), status_code=404)
+    if 'favicon' in url:
+        send_header(client, status_code=404)
+    else:
+        send_header(client, status_code=307, redirect='/')
```

Comment:
can you explain why this is an improvement?

Reply:
I see now, that this would only be necessary if a custom DNS where in use, but like this it is only a very minor improvement of 404 handling and even that is debatable.

Reply:
not sure what you mean.

but in general, a simple and direct error handling is preferable (just say there is a problem and what the problem is). if one introduces some magic instead of that, it often makes finding problems more complicated, hides problems.

Reply:
Proceed as you wish.